HopsanGenerator/src/generators/HopsanGeneratorBase.cpp

```diff
@@ -193,7 +210,7 @@ QString HopsanGeneratorBase::generateSourceCodefromComponentSpec(ComponentSpecif
         QString id = QString::number(portId);
         if(comp.portNodeTypes[i] == "NodeSignal")
         {
-            allVarNames << comp.portNames[i]+"__y";
+            allVarNames << "*mp"+comp.portNames[i]+"";
```

Comment:
This variable now has slightly confusing name (allVarNames), it is no longer variable names, but actually part of the declaration, since * is part of the "name"